### PR TITLE
Parse corrigendum

### LINKED ISSUE #15: Parse PubID with relationships (Corrigendum)
Some PubIDs indicate a technical correction on a previous edition of the document. These are identified in the ID with `/Cor n-yyyy` where `n` is a serial number of the corrigendum, and `yyyy` the year of publishing of the corrigendum.

```
ANSI C63.5-2017/Cor 1-2019 (Corrigendum to ANSI C63.5-2017)
IEEE 1672-2006/Cor 1-2008 (Corrigendum to IEEE Std 1672-2006)
IEEE STD 1137-1991/Cor1-2009
```

### PR DESCRIPTION
**Pull request by mico**
closes #15

### CODE REVIEW

File: spec/pubid_ieee/identifier_spec.rb [hunk 1]
```diff
@@ -480,8 +480,22 @@ def context_description(ex)
     it_behaves_like "converts pubid to pubid"
   end
 
-  context "IEEE Std 802.11ae-2012" do
-    let(:pubid) { "IEEE Std 802.11ae-2012" }
+  context "IEEE Std 1003.1-2001/Cor 2-2004" do
+    let(:pubid) { "IEEE Std 1003.1-2001/Cor 2-2004" }
```
Comment (ronaldtse) [inline spec/pubid_ieee/identifier_spec.rb:484]: Why does this have a space between `Cor 2` but the others are like `Cor1`?

Comment (mico) [reply spec/pubid_ieee/identifier_spec.rb:484]: > Why does this have a space between `Cor 2` but the others are like `Cor1`?

There are many ways of how they can be represented:
```
IEEE Std 802.1Q-2005/Cor1-2008 (Corrigendum to IEEE Std 802.1Q-2005)
IEEE Std C62.41.2-2002_Cor 1-2012 (Corrigendum to IEEE Std C62.41.2-2002)
IEEE PC57.12.90-2015/Cor1/D3.1, September 2016
IEEE P802.1AB-Cor-1/D2.0, January 2013
IEEE Std 535-2013/Cor. 1-2017 (Corrigenda of IEEE Std 535-2013)

```

File: spec/fixtures/pubid-parsed.txt [hunk 2]
```diff
@@ -7957,3 +7957,189 @@ IEEE Std 802.11ah-2016 (Amendment to IEEE Std 802.11-2016, as amended by IEEE St
...
+IEEE P802.1AS-Cor-1/D3.0, January 2013
+IEEE P802.1AS-Cor-1/D3.1, April 2013
+IEEE P802.1AS-Cor-2/D2.0, March 2015
+IEEE P802.1Q_Cor-2/D2.1, August 2012
+IEEE Std 497-2002/Cor 1-2007(Corrigendum to IEEE Std 497-2002)
+IEEE Std 802.15.4x-2019 (Amendment to IEEE 802.15.4-2015 as amended by IEEE 802.15.4n-2016, IEEE 802.15.4q-2016, IEEE 802.15.4u-2016, IEEE 802.15.4t-2017, IEEE 802.15.4v-2017, IEEE 802.15.4s-2018, and IEEE 802.15.4-2015/Cor. 1-2018)
+IEEE Std 535-2013/Cor. 1-2017 (Corrigenda of IEEE Std 535-2013)
+IEEE Std 802.1AC-2016/Cor 1-2018 (Corrigenda to IEEE Std 802.1AC-2016)
+IEEE Std C37.41-2016/Cor 1-2017 (Corrigenda to IEEE Std C37.41-2016)
+IEEE Std C95.1-2019/Cor2-2020 (Corrigenda to IEEE Std C95.1-2019)
```
Comment (mico) [inline spec/fixtures/pubid-parsed.txt:8145]: @ronaldtse here you can see all identifiers for corrigendum we can parse

File: spec/pubid_ieee/identifier_spec.rb (hunk 1 above)
Comment (ronaldtse) [reply spec/pubid_ieee/identifier_spec.rb:484]: I found these patterns:

```
grep -e 'Cor\.' pubid-to-parse.txt | wc -l
      10
grep -e 'Cor\d' pubid-to-parse.txt | wc -l
      90
grep -e 'Cor \d' pubid-to-parse.txt | wc -l
     131
```

Out of them the last one is most popular.

Let's use `Cor \d` (and `Cor \d-yyyy`) as the pattern.

**Review by ronaldtse — APPROVED** [review]
Thank you @mico !

**Comment by ronaldtse** [comment]
@mico could you help rebase and merge? Thanks!

**Pull request merged**